operator/main.go

```diff
@@ -124,6 +119,17 @@ func main() {
 	setupManager(flagVar, controllers.NewCacheFunc(), scheme)
 }
 
+func configLogger() logr.Logger {
+	atomicLevel := zap.NewAtomicLevel()
+	atomicLevel.SetLevel(zap.DebugLevel)
+	encoderConfig := zap.NewProductionEncoderConfig()
+	encoderConfig.TimeKey = "timestamp"
+	encoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
```

Comment:
Why not use RFC3339?

Reply:
RFC3339     = "2006-01-02T15:04:05Z07:00"
RFC3339Nano = "2006-01-02T15:04:05.999999999Z07:00"
	
The standard RFC3339 only precise to second, we can use RFC3339Nano

Reply:
RFC Nano is implementation and os-dependant and vary with precision. Also I do not think we need nano level precision on controllers.

Reply:
I think at least we need millisecond level, first is the reconciliation is really fast, I think it's better to have distinguish for each reconciliation in time level, especially when the situation happens when we have many kyma and concurrent workers configured. btw, this is the default timestamp of the runtime controller after convert with k9s, it's by default use nana level timestamp. within 1 second, we currently have hundreds of log messages in load test cluster.

Regarding os-dependant, which operation system not support it? But the operator running inside k8s container with linux core, should be fine, or?
```
│ 2022-11-03T14:22:38.510444813Z manager 1.6674853585103135e+09    INFO    reconciling modules    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--34-171","namespace":"default"}, "namespace": "default", "name": "kyma--34-171", │
│ 2022-11-03T14:22:38.510478085Z manager 1.6674853585103347e+09    INFO    reconciling modules    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--42-26","namespace":"default"}, "namespace": "default", "name": "kyma--42-26", " │
│ 2022-11-03T14:22:38.511603205Z manager 1.6674853585114877e+09    INFO    reconciling modules    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--18-100","namespace":"default"}, "namespace": "default", "name": "kyma--18-100", │
│ 2022-11-03T14:22:38.511660618Z manager 1.6674853585115814e+09    INFO    reconciling modules    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--33-4","namespace":"default"}, "namespace": "default", "name": "kyma--33-4", "re │
│ 2022-11-03T14:22:38.526287964Z manager 1.6674853585261514e+09    INFO    reconciling modules    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--4-186","namespace":"default"}, "namespace": "default", "name": "kyma--4-186", " │
│ 2022-11-03T14:22:38.551514769Z manager 1.6674853585513988e+09    INFO    reconciling modules    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--45-51","namespace":"default"}, "namespace": "default", "name": "kyma--45-51", " │
│ 2022-11-03T14:22:38.561130942Z manager 1.667485358561005e+09    INFO    syncing state    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--18-100","namespace":"default"}, "namespace": "default", "name": "kyma--18-100", "recon │
│ 2022-11-03T14:22:38.564486954Z manager 1.6674853585643847e+09    INFO    syncing state    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--33-4","namespace":"default"}, "namespace": "default", "name": "kyma--33-4", "reconcil │
│ 2022-11-03T14:22:38.565699159Z manager 1.6674853585655954e+09    INFO    syncing state    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--34-171","namespace":"default"}, "namespace": "default", "name": "kyma--34-171", "reco │
│ 2022-11-03T14:22:38.568483257Z manager 1.6674853585683784e+09    INFO    syncing state    {"controller": "kyma", "controllerGroup": "operator.kyma-project.io", "controllerKind": "Kyma", "Kyma": {"name":"kyma--42-26","namespace":"default"}, "namespace": "default", "name": "kyma--42-26", "reconc │